Дз 22 спринта

ZarubaGit opened the pull request:
1. Создал плейлист
2. Отобразил списко плейлистов
3. Реализация BottomSheet на экране AudioPlayer

ZarubaGit committed:
Дз 22 спринта
1. Создал плейлист
2. Отобразил списко плейлистов
3. Реализация BottomSheet на экране AudioPlayer

ZarubaGit committed:
Исправил отображение цвета индикатора, при вводе текста на экране fragment_add_play_list.xml

kosstyango commented:
🍏 Рекомендации, не влияющие на результат проверки работы:
Курсор в поле поиска должен быть синий в обеих темах, а у тебя он - белый в темной теме и черный - в сетлой теме
Нажатие на кнопку "Очистить поиск" очищает поисковый запрос, но НЕ скрывает клавиатуру.
Треки с ОЧЕНЬ длинным названием альбома показываются неправильно: длительность переносится на след.строку (см.скрин)
В диалоге "Завершить создание плейлиста" кнопки "Отмена" и "Завершить" в светлой теме показаны белым шрифтом на светлом фоне - их плохо видно. А в тёмной теме этот диалог практически не виден: Темно-серый фон и чёрные буквы...

![Дефект прошлых спринтов](https://github.com/ZarubaGit/PlaylistMaker/assets/73995013/746b7994-b8ed-4b35-8332-1b3620897b8c)

ZarubaGit committed:
Исправил визуаьные недочеты.
1. Диологовое окно неправильно отображало цвет элементов в зависимости от темы приложения
2. Исправил неправильное отображение строки с именем исполнителя
